django/contrib/gis/templates/gis/openlayers.html

```diff
@@ -1,27 +1,30 @@
-{% load i18n l10n static %}
+{% load i18n l10n %}
 <style type="text/css">{% block map_css %}{% get_current_language_bidi as LANGUAGE_BIDI %}
     #{{ id }}_map { width: {{ map_width }}px; height: {{ map_height }}px; }
     #{{ id }}_map .aligned label { float: inherit; }
     #{{ id }}_div_map { position: relative; vertical-align: top; float: {{ LANGUAGE_BIDI|yesno:"right,left" }}; }
     {% if not display_raw %}#{{ id }} { display: none; }{% endif %}
-    .olControlEditingToolbar .olControlModifyFeatureItemActive {
-        background-image: url("{% static "admin/img/gis/move_vertex_on.svg" %});
-        background-repeat: no-repeat;
-    }
-    .olControlEditingToolbar .olControlModifyFeatureItemInactive {
-        background-image: url("{% static "admin/img/gis/move_vertex_off.svg" %});
-        background-repeat: no-repeat;
-    }{% endblock %}
+    {% endblock %}
 </style>
 
 <div id="{{ id }}_div_map">
     <div id="{{ id }}_map"></div>
-    <span class="clear_features"><a href="javascript:{{ module }}.clearFeatures()">{% trans "Delete all Features" %}</a></span>
+    {% if not disabled %}<span class="clear_features"><a href="javascript:{{ module }}.clearFeatures()">{% trans "Delete all Features" %}</a></span>{% endif %}
     {% if display_raw %}<p>{% trans "Debugging window (serialized value)" %}</p>{% endif %}
     <textarea id="{{ id }}" class="vSerializedField required" cols="150" rows="10" name="{{ name }}">{{ serialized }}</textarea>
     <script type="text/javascript">
         {% block map_options %}var map_options = {};{% endblock %}
+        {% block base_layer %}
+            var base_layer = new ol.layer.Tile({
+                title: "OpenLayers WMS",
+                source: new ol.source.TileWMS({
+                    url: 'http://demo.opengeo.org/geoserver/wms',
```

Comment:
Just use the NASA GBIF instead. No WMS but WMTS you can consume like OSM tiles but provide the same imagery. The end point should not die as maintained by the NASA.
Just look at full illustrated example below:

```
<!DOCTYPE html>
<html>
  <head>
    <title>Simple Map with GBIF</title>
    <link rel="stylesheet" href="http://openlayers.org/en/v3.15.1/css/ol.css" type="text/css">
    <script src="http://openlayers.org/en/v3.15.1/build/ol.js"></script>
  </head>
  <body>
    <div id="map" class="map"></div>
    <script>
      var source = new ol.source.XYZ({
          url: "http://map1{a-c}.vis.earthdata.nasa.gov/wmts/" +
              "epsg3857/best/BlueMarble_ShadedRelief_Bathymetry/default/%7BTime%7D/" +
              "GoogleMapsCompatible_Level8/{z}/{y}/{x}.jpg"
      });
      var map = new ol.Map({
        layers: [
          new ol.layer.Tile({
            source: source
          })
        ],
        target: 'map',
        view: new ol.View({
          center: [0, 0],
          zoom: 2,
          maxZoom: 8
        })
      });
    </script>
  </body>
</html>
```
If you want to try out other default layer from GBIF, go to https://wiki.earthdata.nasa.gov/display/GIBS/GIBS+API+for+Developers